schema/spacewalk/common/tables/suseClusters.sql

```diff
@@ -0,0 +1,48 @@
+-- Copyright (c) 2020 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+-- Red Hat trademarks are not licensed under GPLv2. No permission is
+-- granted to use or replicate Red Hat trademarks that are incorporated
+-- in this software or its documentation.
+--
+
+CREATE TABLE suseClusters (
+    id          NUMERIC NOT NULL
+                    CONSTRAINT suse_clusters_id_pk PRIMARY KEY,
+    org_id      NUMERIC NOT NULL
+                    CONSTRAINT rhn_server_oid_fk
+                    REFERENCES web_customer (id)
+                    ON DELETE CASCADE,
+    label       VARCHAR(256) NOT NULL,
+    name        VARCHAR(256) NOT NULL,
+    description VARCHAR(4096) DEFAULT '',
+    provider    VARCHAR(50) NOT NULL,
+    management_node_id  NUMERIC
+                        CONSTRAINT suse_clusters_mgmt_node_fk
+                        REFERENCES rhnServer (id)
+                        ON DELETE SET NULL,
+    group_id    NUMERIC NOT NULL
+                    CONSTRAINT suse_clusters_group_fk
+                    REFERENCES rhnServerGroup (id),
```

Comment:
Is it wanted to not have an `ON DELETE` statement here?

Reply:
Yes, it's intentional. The cluster group must be delete only together with the cluster. I added [this commit](https://github.com/uyuni-project/uyuni/pull/2070/commits/acb1326ab7092d7661c283b58bc8a53b369abb22) to disable the delete button in the group UI and to throw an exception when deleting from the API.

Reply:
Ok